dev/test/query.ts

```diff
@@ -282,27 +285,47 @@ function endAt(
   return {endAt: cursor};
 }
 
+export function queryEqualsWithParent(
+  actual: api.IRunQueryRequest | undefined,
+  parent: string,
+  ...protoComponents: api.IStructuredQuery[]
+): void {
+  queryEqualsHelper(actual, '/' + parent, ...protoComponents);
+}
+
 export function queryEquals(
   actual: api.IRunQueryRequest | undefined,
   ...protoComponents: api.IStructuredQuery[]
-) {
+): void {
+  queryEqualsHelper(actual, '', ...protoComponents);
+}
+
+function queryEqualsHelper(
```

Comment:
I am not a huge fan of `Helpers`. Would it be possible to just make this `queryEqualsWithParent`?

Reply:
You could do:

```
export function queryEquals(
  actual: api.IRunQueryRequest | undefined,
  parentOrProtoComponent: string|api.IStructuredQuery|undefined,
  ...protoComponents: api.IStructuredQuery[]
): void;
```

But that is not all that pretty.

I was actually thinking you keep both `queryEquals` and `queryEqualsWithParent`, but merge `queryEqualsWithParent` and `queryEqualsHelper`.